server/bin/index.ts

```diff
@@ -125,7 +125,10 @@ function main(): void {
     orchestrator
       .run(filePath)
       .then((doc: Document) => {
-        const nbTexts = doc.pages.map(p => p.elements.length).reduce((a, b) => a + b, 0);
+        // TODO: when we start OCRing all the images inside a pdf, the part '- p.getElementsOfType(Image).length'
+        // should be removed
+        const nbTexts = doc.pages.map(p => p.elements.length - p.getElementsOfType(Image).length)
```

Comment:
Hi @aarohijohal 
Could be better this approach ?? And remove the TODO ??
`const nbTexts = doc.getElementsOfType(Word, true).length;`

Then when we start to OCRing images found in text based pdf we will don't need to update this code, right ?

Reply:
Excellent suggestion! But we could always have a document without any words and only tables containing barcodes for example.

Reply:
As we commented please try to find best & understandable code to detect image based pds's

Reply:
Done in ee62014